### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by FluxFlu**
Seemed to be a typo. Hope this helps.

### CODE REVIEW

File: examples/textures/textures_image_drawing.c [hunk 1]
```diff
@@ -93,4 +93,4 @@ int main(void)
     //--------------------------------------------------------------------------------------
 
     return 0;
-}
```
Comment (FluxFlu) [inline examples/textures/textures_image_drawing.c:96]: Not sure why this shows up in the diff.

Comment (raysan5) [reply examples/textures/textures_image_drawing.c:96]: It seems the close brake has been deleted, please, could you review it?

Comment (CrackedPixel) [reply examples/textures/textures_image_drawing.c:96]: > It seems the close brake has been deleted, please, could you review it?

the brace was unaffected. github is showing that the file originally did not have an empty line at the end, and now one has been added. im guessing @FluxFlu has some linter program running that did it automatically

**Comment by raysan5** [comment]
@FluxFlu @CrackedPixel thanks for the review!

**Pull request merged**